update-serve-file.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
I wish we could use composer. We should figure out how to manage dependencies with it properly.